### DISCUSSION TITLE
Android Songs folder location

### DISCUSSION THREAD

**Discussion by megoodosu** [ppy/osu#16289] [Q&A] [answered]
Hi everyone, so the tldr; is that i cant seem to find the osu!lazer songs folder in Android.

Here's the explonation why i need to find it;
I started up osu!lazer fresh start, my maps were gone.
I taught it read from osu!droid or opsu! so i tried to do that and, it didnt work, didnt read the Songs folders. 
I was like:
"Hmmm maybe it has its OWN Songs folder, because we don't have stable on Android".
I checked inside Android/data/sh.ppy.osulazer and I didnt find any Songs folder related thing.

|Is there a way to find the Songs folder in Android and export/import maps to it?

**Comment by Theighlin** [comment, marked as answer]
There is no "Songs" folder for osu!lazer. You can open osk files via the game or you can use the in game beatmap listing to download beatmaps directly
